pgrx-sql-entity-graph/src/used_type.rs

```diff
@@ -608,6 +613,146 @@ fn resolve_option_inner(
     }
 }
 
+fn resolve_result_inner(
+    original: syn::TypePath,
+) -> syn::Result<(syn::Type, Option<CompositeTypeMacro>)> {
+    let segments = &original.path;
+    let last = segments
+        .segments
+        .last()
+        .ok_or(syn::Error::new(original.span(), "Could not read last segment of path"))?;
+
+    // Get the path of our Result type, to handle crate::module::Result pattern
+    // TODO - more robust / idiomatic way of handling this?
```

Comment:
Robust against?

Reply:
I wasn't fully understanding the design of UsedType's codegen at this time, and I was making some incorrect guesses about how this would fit together.  I can remove this comment.

Reply:
I prefer more actionable TODOs, yes.

Reply:
removed in b2555d0